tests/integration/ceph_multi_cluster_test.go

```diff
@@ -142,8 +142,7 @@ func NewMCTestOperations(t func() *testing.T, namespace1 string, namespace2 stri
 	i := installer.NewCephInstaller(t, kh.Clientset, false, "", installer.VersionMaster, installer.NautilusVersion, cleanupHost)
 
 	op := &MCTestOperations{i, kh, t, namespace1, namespace2, installer.SystemNamespace(namespace1), "", false}
-	present, err := kh.IsStorageClassPresent("manual")
-	if present && kh.VersionAtLeast("v1.13.0") {
+	if kh.VersionAtLeast("v1.13.0") {
```

Comment:
When is the SC created?

Reply:
> When is the SC created?

It's in the `setup()` of the multi-cluster test. Here `localPathPVCmd` creates the SC. Since this command is always executed if `o.testOverPVC` is true, `present` check is not necessary.  

https://github.com/rook/rook/blob/43296ef36acdf38d5bc3d524d70ce3a92c8eeee3/tests/integration/ceph_multi_cluster_test.go#L157

Reply:
> When is the SC created?

It's in the `setup()` of the multi-cluster test. Here `localPathPVCmd` creates the SC. Since this command is always executed if `o.testOverPVC` is true, `present` check is not necessary.  

https://github.com/rook/rook/blob/43296ef36acdf38d5bc3d524d70ce3a92c8eeee3/tests/integration/ceph_multi_cluster_test.go#L157